scripts/mcp_common.sh

```
@@ -2193,82 +2199,71 @@ install_mcp_mail() {
         log_with_timestamp "Server mcp_mail already exists, skipping"
         INSTALL_RESULTS["mcp_mail"]="ALREADY_EXISTS"
         SUCCESSFUL_INSTALLS=$((SUCCESSFUL_INSTALLS + 1))
-        return 0
-    fi
+    else
+        # MCP Mail requires the server to be running locally on port 8765
+        # Check if MCP_MAIL_BEARER_TOKEN is set
+        local mcp_mail_token="${MCP_MAIL_BEARER_TOKEN:-}"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)